### PR TITLE
Adding PHP implementation for LIS (Issue #2253)

### PR DESCRIPTION
**Pull request by sukritishah15**
Fixes #2253 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- PHP

#### Files Added:
- LIS.php 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by sukritishah15** [comment]
@raghav-dalmia @SakshiShreya @AdityaShD Kindly review my PR.

File: Longest_Increasing_Subsequence/LIS.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+// PHP Implementation of LIS using Dynamic Programming
```
Comment (asha15) [inline Longest_Increasing_Subsequence/LIS.php:2]: Put this commend at the beginning of the file

File: Longest_Increasing_Subsequence/LIS.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+// fumction to return the length of LIS
+function lisLen($arr, $n){
+    // $dp[$i] represents length of LIS ending at index i
```
Comment (asha15) [inline Longest_Increasing_Subsequence/LIS.php:6]: remove this commend

Comment (sukritishah15) [reply Longest_Increasing_Subsequence/LIS.php]: Made the second required change in the PR. Left the first one as discussed with you @asha15 
Kindly review it.

**Comment by sukritishah15** [comment]
Since, it has been approved, kindly merge it @asha15 @jainaman224

**Pull request merged**